docs/site/angular-auth-oidc-client/docs/documentation/public-api.md

```diff
@@ -293,6 +293,20 @@ const refreshToken = this.oidcSecurityService.getRefreshToken();
 const refreshToken = this.oidcSecurityService.getRefreshToken('configId');
 ```
 
+## getAuthenticationResult(configId?: string)
+
+Returns the authentication result, if present, for the sign-in.
+@param configId The configId to check the information for. If not passed, the first configs will be taken

```

Comment:
as this is a public api md remove the `@param` and `@returns` statements and write a human readable nice message :)

Reply:
fixed